packages/core/src/DocsUtils/Section/Section.tsx

```diff
@@ -1,7 +1,7 @@
-import kebabCase from 'lodash/kebabCase'
 import React from 'react'
-
-import { Box, Text } from '../..'
```

Comment:
(n-b) More barrels to clear out?

Reply:
yea i left DocsUtils alone the last time, still think it might be a separate package soon